frontend/src/components/pages/BratPage.tsx

```diff
@@ -0,0 +1,6 @@
+export const BratPage = () => {
+  return <div className="bg-[#89CC04] w-full flex-col gap-8 font-['Arial'] text-9xl antialiased blur-[2px] h-full flex items-center justify-center">
+    <h1>online</h1>
+    <h1>is brat</h1>
+  </div>
+}
```

Comment:
Sikkert pirk, men jeg liker å droppe `return` i komponenter som kun skal returnere html:

```suggestion
export const BratPage = () => (
  <div className="bg-[#89CC04] w-full flex-col gap-8 font-['Arial'] text-9xl antialiased blur-[2px] h-full flex items-center justify-center">
    <h1>online</h1>
    <h1>is brat</h1>
  </div>
)
```